server/community.go

```diff
@@ -95,14 +97,25 @@ func (s *Server) assignGreetingLabels(ctx context.Context, pr *model.PullRequest
 		return nil
 	}
 
-	// Is repo configured to setup labels for non-member PRs?
-	if len(repo.GreetingLabels) == 0 {
-		return nil
+	// Check if the repository has the contriutor label already
+	labels := repo.GreetingLabels
```

Comment:
this is a config for each repo, this does not come from the PR.

I think in this function we need to check if the PR submitter is a member of the org and if not we add the label. and ignore if the `GreetingLabels` is configured or not.

WDTY?

Reply:
Yep it is checked (line 96). But instead of ignoring the GreetinLabels, what the patch does is append the `Contributor` label to the []GreetingLabels that the repo already has configured. 

TBH, it does not feel 100% right to hardcode the addition of the label to all PRs. I'd like to have control if I don't want this happening on some repos. Perhaps it could be a flag in the repo configuration?

Reply:
all good! thanks for the clarification